src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -232,6 +233,25 @@
     @end
 @end
 
+@private grpcStreamingDescriptorFunction(xapiClass)
+    @if xapiClass.hasGrpcStreamingMethods
+        private static function getGrpcStreamingDescriptors()
+        {
+            return [
+                @join descriptor : xapiClass.grpcStreamingDescriptors
+                    '{@descriptor.methodName}' => [
+                        'grpcStreamingType' => '{@descriptor.grpcStreamingType.toString}',
```

Comment:
You don't need the `.toString`, the snippet language will do that automatically.